arviz/plots/styles/arviz-doc.mplstyle

```diff
@@ -30,16 +30,9 @@ axes.facecolor:         white
 axes.edgecolor:         .33         # axes edge color
 axes.linewidth:         0.8         # edge line width
 
-axes.grid:              True        # show grid
-# axes.grid.axis:         y        # which axis the grid should apply to
-axes.grid.which:        major       # grid lines at {major, minor, both} ticks
```

Comment:
Can we keep this? Without explicitly setting `axes.grid.which` to "major", the axis will show both major and minor. I like things more simple and less clutterry, which is what I was hoping this style goes for.

I think the only differentiator between `arviz-doc` and `arviz-docgrid` is the presence of the grid. Even with arviz-doc style applied, you can override the grid visibility by applying `plt.grid()`. If the user happens to do that, then I want it to match `arviz-docgrid`'s style to a tee.